src/store/modules/auth.js

```diff
@@ -67,37 +67,37 @@ const getters = {
 
 const actions = {
   async setAccessToken({ commit, dispatch }, accessToken) {
-    await commit("setAccessToken", accessToken);
+    commit("setAccessToken", accessToken);
     await dispatch("fetchAndUpdateUser");
   },
-  async unsetAccessToken({ commit, dispatch }) {
-    await commit("unsetAccessToken");
+  unsetAccessToken({ commit, dispatch }) {
+    commit("unsetAccessToken");
     dispatch("unsetUser");
     dispatch("unsetAnalyticsAccessToken");
   },
-  async setUser({ commit }, user) {
-    await commit("setUser", user);
+  setUser({ commit }, user) {
+    commit("setUser", user);
   },
-  async unsetUser({ commit }) {
-    await commit("unsetUser");
+  unsetUser({ commit }) {
+    commit("unsetUser");
   },
-  async setActiveWorkspace({ commit }, activeWorkspace) {
-    await commit("setActiveWorkspace", activeWorkspace);
+  setActiveWorkspace({ commit }, activeWorkspace) {
+    commit("setActiveWorkspace", activeWorkspace);
   },
-  async unsetActiveWorkspace({ commit }) {
-    await commit("unsetActiveWorkspace");
+  unsetActiveWorkspace({ commit }) {
+    commit("unsetActiveWorkspace");
   },
-  async saveConfig({ commit }, config) {
-    await commit("saveConfig", config);
+  saveConfig({ commit }, config) {
+    commit("saveConfig", config);
   },
-  async setReAuthenticationState({ commit }, state) {
-    await commit("setReAuthenticationState", state);
+  setReAuthenticationState({ commit }, state) {
+    commit("setReAuthenticationState", state);
   },
-  async setReAuthenticationPromise({ commit }, promise) {
-    await commit("setReAuthenticationPromise", promise);
+  setReAuthenticationPromise({ commit }, promise) {
+    commit("setReAuthenticationPromise", promise);
   },
-  async setReAuthenticationPromiseResolver({ commit }, resolver) {
-    await commit("setReAuthenticationPromiseResolver", resolver);
+  setReAuthenticationPromiseResolver({ commit }, resolver) {
+    commit("setReAuthenticationPromiseResolver", resolver);
```

Comment:
have all these functions being tested in the sense that there is no error after removing the async await - as in logically, as things following each of these calls waiting on them to be completed first? if yes, we should not remove the async/awaits right?